apps/drupal-default/particle_theme/templates/block/block--utc-hero-video--default.html.twig

```diff
@@ -0,0 +1,151 @@
+{#
+/**
+ * @file Video Hero!
+ * Implements a video hero block in dark blue
+ *
+ *
+ * Available variables:
+ */
+ *
+ * This template pulls in the variables to add to the cards. 
+ * References the block--utc-card-base.html.twig to create the individual cards in this grid.
+#}
+{%
+  set classes = [
+    'block',
+	'block--layout-builder', 
+	'block--inline-block:utc-hero-video',
+	'block--utc-video-hero',
+    'block--utc-video-hero-default',
```

Comment:
indent looks wrong, not sure if Twig cares as much as YAML does.

Reply:
Twig doesn't care, but I'll fix.